tensorboard/webapp/widgets/data_table/types.ts

```diff
@@ -39,6 +39,7 @@ export enum ColumnHeaderType {
   MEAN = 'MEAN',
   RAW_CHANGE = 'RAW_CHANGE',
   HPARAM = 'HPARAM',
+  CUSTOM = 'CUSTOM',
```

Comment:
I am considering removing COLOR and using CUSTOM for the color columns.

Reply:
I understand the rationale here, but I think I prefer leaving them. I think keeping them declared this way helps readability. Feel free to disagree though.